Respect users' LDFLAGS.
Patch by Arfrever Frehtes Taifersar Arahesis, bug 181441

--- kdm/backend/Makefile.am
+++ kdm/backend/Makefile.am
@@ -1,7 +1,7 @@
 # forcibly remove thread-related defines & flags
 AUTOMAKE_OPTIONS = foreign
 CPPFLAGS = $(USER_INCLUDES) $(X_INCLUDES) $(KRB4_INCS) $(KRB5_INCS) -I.. -I../..
-LDFLAGS = $(USER_LDFLAGS) $(X_LDFLAGS) $(X_RPATH) $(KRB4_RPATH) $(KRB5_RPATH)
+LDFLAGS = @LDFLAGS@ $(USER_LDFLAGS) $(X_LDFLAGS) $(X_RPATH) $(KRB4_RPATH) $(KRB5_RPATH)
 LDADD = $(LIB_X11) -lXau $(LIBXDMCP) $(PASSWDLIBS) $(LIBSHADOW) $(LIBGEN) \
     $(LIB_LIBS) $(KRB4_LIBS) $(KRB5_LIBS) $(LIBSOCKET) $(LIBRESOLV) \
     $(LIBUCB) $(LIBUTIL) $(LIBPOSIX4)
